src/android/mod.rs

```diff
@@ -67,7 +67,11 @@ pub fn plat_getprop(_: &str, property_info: *const c_void) -> Option<String> {
     if !property_info.is_null() {
         unsafe { __system_property_read_callback(property_info, property_callback, &mut *result) };
     }
-    Some((*result).value)
+
+    match (*result).value {
+        value if !value.is_empty() => Some(value),
```

Comment:
That's really long ago now, but I don't think this is correct.

That is something you might need to wrap like here if value is empty [see here](https://cs.android.com/android/platform/superproject/main/+/main:system/libbase/properties.cpp;l=130;drc=57365e2258cfe0dfb2d26c9cf056075d5a26c573)

AFAIR option here is used for the sake reason to show the property is initialized by a callback (newer bionic versions)